Markdown/cmm-2.0.14/release-notes/2-restrictions.md

```diff
@@ -47,7 +47,17 @@ project in OpenStack Horizon and then try to access Kibana, the Kibana dashboard
 be not loading properly.
 
 
-### Workaround:
+* Workaround:
+  For all browser versions suppported: Please use the default options of your browser to reload the page.
 
-For all browser versions suppported: Please use the default options of your browser to reload the page.
 
+### Filtering more than one element in Kibana using Firefox
```

Comment:
Use capitalized heading style, should be
### Filtering More Than One Element in Kibana Using Firefox

https://www.webucator.com/how-to/how-capitalize-headings-titles.cfm

Reply:
Done